packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -210,10 +210,16 @@ class InkResponse extends StatefulWidget {
     this.splashFactory,
     this.enableFeedback = true,
     this.excludeFromSemantics = false,
+    this.focusNode,
```

Comment:
[we talked this over and decided to leave the API as is]

Just FTR: there's tension here between convenience and separation of concerns. We're exposing the most useful Focus widget parameters here and in other components for the sake of convenience. It's for the not uncommon case where the developer wants to be able to drive the focus directly to this inkwell (or button etc). Unfortunately that means blending the FocusWidget's parameters into other widget configurations. There doesn't seem to be a clean way to avoid this:
 - A TransitionBuilder valued `buildFocus` parameter could be used instead. It would be easy enough to have it return a widget that DTRT, but somewhat clunkier.
- The developer could just wrap the component's child with a Focus widget. But that would create a nested Focus widget (under the one created by the component by default) which might not be what's wanted (Semantics).